tools/cli/src/mentat_cli/command_parser.rs

```diff
@@ -40,8 +42,9 @@ impl Command {
 }
 
 pub fn command(s: &str) -> Command {
+    let skip_spaces = skip_many(space());
```

Comment:
This is the `spaces()` parser from `combine::char`.  I've found that `let p = || ...` and then `p()` is more natural than cloning the parser.